commit e7670df5fbfa09c7bffaf0fb8149c4b6ecb8139f
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-01T21:53:21+00:00 (7 months ago)
    
    TOOLS-2192 buildimage target fails with non-default BUILDIMAGE_VERSION value

diff --git a/tools/mk/Makefile.targ b/tools/mk/Makefile.targ
index defbe4c..d803c07 100644
--- a/tools/mk/Makefile.targ
+++ b/tools/mk/Makefile.targ
@@ -439,10 +439,10 @@ buildimage: stamp-buildimage-prep release $(AGENTS:%=%-prebuilt)
 		-m '$(BUILDIMAGE_MF)' \
 		$(BUILDIMAGE_PKGSRC_ARGS) \
 		-P $(NAME)-zfs
-	cp /tmp/$(NAME)-zfs-$(STAMP).zfs.gz $(ENGBLD_BITS_DIR)/$(NAME)
-	cp /tmp/$(NAME)-zfs-$(STAMP).imgmanifest $(ENGBLD_BITS_DIR)/$(NAME)
-	pfexec rm /tmp/$(NAME)-zfs-$(STAMP).zfs.gz
-	pfexec rm /tmp/$(NAME)-zfs-$(STAMP).imgmanifest
+	cp /tmp/$(NAME)-zfs-$(BUILDIMAGE_VERSION).zfs.gz $(ENGBLD_BITS_DIR)/$(NAME)
+	cp /tmp/$(NAME)-zfs-$(BUILDIMAGE_VERSION).imgmanifest $(ENGBLD_BITS_DIR)/$(NAME)
+	pfexec rm /tmp/$(NAME)-zfs-$(BUILDIMAGE_VERSION).zfs.gz
+	pfexec rm /tmp/$(NAME)-zfs-$(BUILDIMAGE_VERSION).imgmanifest
 	pfexec rm -rf $(BUILDIMAGE_STAGEDIR)
 
 #
